### DISCUSSION TITLE
Feature Request: Collapse/Expand All Blocks of Type in Matrix

### DISCUSSION THREAD

**Discussion by proimage** [craftcms/cms#8588] [Ideas]
It would be useful to be able to have "[Expand/Collapse] all _____ blocks" in the context menu of Matrix blocks.

I work on a site that uses Matrix to build pages modularly. It has a lot (...a _LOT_) of "Spacer" blocks in most pages. I'd love to be able to click the cog on a Spacer block and select "Collapse all Spacer blocks" from the menu.

Thanks!

**Comment by narration-sd** [comment]
Maybe the need fits with a Solo button like this?  #3116

There are some pictures which get the action across along the comments.

**Comment by brandonkelly** [comment]
Duplicate of #1880

**Comment by proimage** [comment]
Just pointing out that this is not _exactly_ a duplicate, as this is about selectively collapsing all blocks _of a specific type_. :)

**Comment by brandonkelly** [comment]
Sorry about that!

**Comment by proimage** [comment]
Just ran across this issue during a search. I've noticed that at some point you started adding labels to various issues to organize them, and saw that this issue seems to have been before that time. SO this is a simple bump so you can add whatever labels—if any—you deem appropriate. :)
